Extensions/CodeGeneration/CodeGenerationDisplayDriver.cs

```diff
@@ -134,14 +134,14 @@ private string ConvertJToken(JToken jToken)
 
                 case JArray jArray:
                     var token = string.Join(", ", jArray.Select(ConvertJToken));
-                    var format = token.Contains("ListValueOption") ? "\n\t\t\t" : "";
+                    var format = token.Contains("ListValueOption") ? "\n    " : "";

```

Comment:
> Warning	CA1307	The behavior of 'string.Contains(string)' could vary based on the current user's locale settings. Replace this call in 'Lombiq.HelpfulExtensions.Extensions.CodeGeneration.CodeGenerationDisplayDriver.ConvertJToken(Newtonsoft.Json.Linq.JToken)' with a call to 'string.Contains(string, System.StringComparison)'.

Reply:
I should use `StringComparison.InvariantCulture` or `StringComparison.InvariantCultureIgnoreCase`?

Reply:
Rather `ContainsOrdinalIgnoreCase()` from Helpful Libraries.